internal/elasticsearch/node_stats/stats_api.go

```diff
@@ -0,0 +1,145 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package node_stats
+
+import (
+	"encoding/json"
+	"io"
+	"strings"
+
+	"github.com/elastic/go-elasticsearch/v7"
+	"github.com/pkg/errors"
+
+	es "github.com/elastic/elastic-package/internal/elasticsearch"
```

Comment:
Trying to reduce the need of these `es` aliases in https://github.com/elastic/elastic-package/pull/596.

Reply:
done